apps/dg/components/case_card/column_resize_handle.js

```diff
@@ -0,0 +1,146 @@
+          deltaStart = e.pageX - this.state.resizeWidth;
+          this.setState({ isResizing: true });
+        },
+
+        continueResize: function(e) {
+          var newWidth = e.pageX - deltaStart,
+              minWidth = this.props.minWidth || 20,
+              maxWidth = this.props.containerWidth - minWidth;
+          newWidth = Math.max(newWidth, minWidth);
+          newWidth = Math.min(newWidth, maxWidth);
+          deltaStart = e.pageX - newWidth;
+          if (newWidth !== this.state.resizeWidth) {
+            this.props.onResize && this.props.onResize(newWidth);
+            this.setState({ resizeWidth: newWidth });
+          }
+        },
+
+        endResize: function() {
+          this.setState({ isResizing: false });
+        },
+
+        handleMouseDown: function(e) {
+          if (!this.props.enabled || (e.button !== 0)) return;
+          this.beginResize(e);
+          document.addEventListener('mousemove', this.handleMouseMove, true);
+          document.addEventListener('mouseup', this.handleMouseUp, true);
+          stop(e);
+        },
+
+        handleMouseMove: function(e) {
+          this.continueResize(e);
+          stop(e);
+        },
+
+        handleMouseUp: function(e) {
+          this.endResize();
+          document.removeEventListener('mousemove', this.handleMouseMove, true);
+          document.removeEventListener('mouseup', this.handleMouseUp, true);
+          stop(e);
+        },
+
+        handleInitialTouch: function(e) {
+          if (!this.props.enabled) return;
+
+          var touch = e.changedTouches[0];
+          resizeTouchId = touch.identifier;
+          this.beginResize(touch);
+
+          // An individual 'touchstart' or 'touchmove' may be for some other touch, so we
+          // don't care about the type of event, only whether the resize touch is still active.
+          document.addEventListener('touchstart', this.handleSubsequentTouch, true);
+          document.addEventListener('touchmove', this.handleSubsequentTouch, true);
+          document.addEventListener('touchcancel', this.handleSubsequentTouch, true);
+          document.addEventListener('touchend', this.handleSubsequentTouch, true);
+          stop(e);
+        },
+
+        handleSubsequentTouch: function(e) {
+          for (var i = 0; i < e.touches.length; ++i) {
+            if (e.touches[i].identifier === resizeTouchId) {
+              // resize touch is still active
+              this.continueResize(e.touches[i]);
+              return;
+            }
+          }
+          // resize touch is no longer active
+          this.endResize();
+
+          document.removeEventListener('touchstart', this.handleSubsequentTouch, true);
+          document.removeEventListener('touchmove', this.handleSubsequentTouch, true);
+          document.removeEventListener('touchcancel', this.handleSubsequentTouch, true);
+          document.removeEventListener('touchend', this.handleSubsequentTouch, true);
```

Comment:
Note that we don't `stop()` the subsequent touch events, because it's likely that they're destined for other valid handlers. This allows multiple touches to be active simultaneously. For a good time, try resizing a case card component while actively resizing the column on a touch device. 😉